play-crate-io/crate.service

```diff
@@ -18,7 +18,7 @@ ExecStart=/bin/bash -c '\
         --volume /mnt/data1/crate:/data \
         --volume /tmp:/tmp \
         --env CRATE_HEAP_SIZE=16g \
-        crate:latest \
+        crate/crate:play \
```

Comment:
@chaudum `:testing` refers to our current testing release. on play.crate.io should only run versions that are getting stable on the same day.